src/components/WhereToBuy/Map/Map.js

```diff
@@ -170,7 +171,7 @@ class Map extends Component {
 							</ Marker>
 							);
 					})}
-					{this.props.showRegions ? ::this.renderRegions() : null}
+					{this.props.showRegions && this.props.regions ? ::this.renderRegions() : null}
```

Comment:
Can you make this explicit, either 

+					{this.props.showRegions && (this.props.regions ? ::this.renderRegions() : null)}
or

+					{(this.props.showRegions && this.props.regions) ? ::this.renderRegions() : null}